HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -54,9 +54,27 @@
+      <sch:assert role='ERROR' test='count(h:BackupHeatingCapacityIncrement) &lt;= 1'>Expected 0 or 1 element(s) for xpath: BackupHeatingCapacityIncrement</sch:assert>
+      <sch:assert role='ERROR' test='number(h:BackupHeatingCapacityIncrement) &gt; 0 or not (h:BackupHeatingCapacityIncrement)'>BackupHeatingCapacityIncrement should be greater than 0.0</sch:assert>
+      <sch:assert role='ERROR' test='number(../h:Timestep) = 1 or not(../h:Timestep) or not(h:BackupHeatingCapacityIncrement)'>Expected ../Timestep to be 1.0</sch:assert>
+      <sch:assert role='ERROR' test='number(../h:Timestep) = 1 or not(../h:Timestep) or not(h:OnOffThermostatDeadbandTemperature)'>Expected ../Timestep to be 1.0</sch:assert>
     </sch:rule>
   </sch:pattern>
   
+  <sch:pattern>
+    <sch:title>[HVACGEBControl=DeadbandControl]</sch:title>
+    <sch:rule context='/h:HPXML/h:SoftwareInfo/h:extension/h:SimulationControl/h:AdvancedResearchFeatures[h:OnOffThermostatDeadbandTemperature]'>
+      <sch:assert role='ERROR' test='count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem[h:FractionCoolLoadServed > 0]) + count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:HeatPump[h:FractionCoolLoadServed > 0]) &lt;= count(../../../../h:Building)'>Expected at maximum one cooling system for each Building</sch:assert>
+      <sch:assert role='ERROR' test='count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:HeatingSystem[h:FractionHeatLoadServed > 0]) + count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:HeatPump[h:FractionHeatLoadServed > 0]) &lt;= count(../../../../h:Building)'>Expected at maximum one heating system for each Building</sch:assert>
+      <sch:assert role='ERROR' test='../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem/h:CoolingSystemType[text()="central air conditioner" or text()="room air conditioner" or text()="packaged terminal air conditioner"] or ../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/h:HeatPump/h:HeatPumpType[text() = "air-to-air"] or count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:FractionCoolLoadServed) = 0'>Unexpected cooling system types, only support dx systems</sch:assert>
+      <sch:assert role='ERROR' test='sum(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:FractionHeatLoadServed) &gt;= 0.99 or count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:FractionHeatLoadServed) = 0'>Expected sum(FractionHeatLoadServed) to be equal to 1</sch:assert>
+      <sch:assert role='ERROR' test='sum(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:FractionCoolLoadServed) &gt;= 0.99 or count(../../../../h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:FractionCoolLoadServed) = 0'>Expected sum(FractionCoolLoadServed) to be equal to 1</sch:assert>
```

Comment:
A lot going on here...
- Checking across all the `Building` elements in the HPXML file is kind of weird and doesn't really do the right thing. If one `Building` has two cooling systems and another `Building` has zero cooling systems, the error-check will pass. It will make more sense to do these checks within each `Building`. Let's discuss.
- Do we really have to throw an error if there's any non-DX heating/cooling system? I don't really understand why it can't apply to non-DX systems, but if indeed it can't, why do we have to throw an error? Can't we just ignore it for those system types (with a warning if it makes sense)?
- The last two checks are supposed to check that the fractions are equal to 1, but all they check is that they are greater than 0.99. There's no upper limit check. Maybe you did this because of when there are multiple `Building` elements, but we shouldn't be skipping important error-checking just to get CI tests to pass. Seems like it could be easily handled. Again, let's discuss.

Reply:
The first thought makes sense, we still need to check the systems at building level instead of a sum. Let me think how to do that.
The reason to only enable with DX system is because we're modeling the cycling behavior for start-up degradation, which only applies to DX system if I recall it correctly.
I didn't recall why I just checked it's greater than 0.99, probably forgot to add the other side checking, let me rethink about it with the building level check.

Reply:
I think a building-level check would use a Schematron context that is something like: `/h:HPXML[h:SoftwareInfo/h:extension/h:SimulationControl/h:AdvancedResearchFeatures/h:OnOffThermostatDeadbandTemperature]/h:Building/h:BuildingDetails`

Reply:
I found that the schema only allows maximum fraction to be 1.0, so if we limit to only allow one cooling system/heat pump then it has to be smaller than 1.0. I was trying to add the other side check but I cannot add test in test_validation.rb due to the schema constraint. So I think the check of 0.99 should be sufficient in this case.

Reply:
For the second item, I'm thinking maybe we can add a method in measure.rb that checks system type and throw error or warning if needed. There're a few checks in hvac.rb, but they're under specific method call (eg. [this for variable speed system](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/hvac.rb#L1850-L1852) but it won't be called when apply_air_source_hvac_systems is not called. Maybe measure.rb is the right place to check the feature availability for those only apply to specific system types)